app/assets/stylesheets/active_admin/components/_breadcrumbs.scss

```diff
@@ -1,20 +1,21 @@
 .breadcrumb {
   display: block;
-  font-size: 0.9em;
-  font-weight: normal;
-  line-height: 1.0em;
-  margin-bottom: 12px;
-  text-transform: uppercase;
+  list-style: none;
+  margin: 0 0 2px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 2px` instead of `0 0 2px 0`